files/nvidia-overclock.sh

```diff
@@ -1,32 +1,51 @@
 #!/usr/bin/env bash
 
-
-# Overclocking with nvidia-settings
-# Increase or decrease in small increments (+/- 25)
-
-
+#Skip manual settings per card.
+SKIP_MANUAL_SETTINGS=true
```

Comment:
I think this could be labelled more clearly e.g. maybe call it USE_INDIVIDUAL_SETTINGS and false by default? Then true goes into the individual settings loop?